spec/request/health_care_applications_request_spec.rb

```diff
@@ -111,10 +111,14 @@
           allow_any_instance_of(HCA::Service).to receive(:post) do
             raise error
           end
+          Settings.sentry.dsn = 'asdf'
+        end
+        after do
+          Settings.sentry.dsn = nil
         end
 
         it 'should render error message' do
-          expect(Raven).to receive(:capture_exception).with(error).once
+          expect(Raven).to receive(:capture_exception).with(error).twice
```

Comment:
I think I may have discovered a bug.  The spec was expecting `Raven.capture_exception` to be called once which passed as a spec, but in production, it was actually being called twice:
1. at health_care_applications_controller.rb:26
2. inside `application_controller#log_error`
The difference being that # 1 didn't depend on ENV['SENTRY_DSN'] like # 2 did.  I have since added a `before` & `after` block to mock out the Sentry DSN setting.  

@lihanli  or @markolson Is being called twice the actual desired behavior? (I ask because you two look like the owners of this spec)